repose-aggregator/src/docs/asciidoc/developer/docker.adoc

```diff
@@ -5,16 +5,16 @@ This is a collection of information cobbled together from several sources, but m
 The rest of this assumes you are already familiar with the contents of the <<../recipes/quick-start.adoc#,Quick Start>> and <<../recipes/docker.adoc#,Docker>> recipes.
 Since you made it here your hunger for *Repose Docker* knowledge must not have been satiated, so read on and enjoy.
 
-== Overview
+== Running Repose as a Docker Container
 Basically you just need to:
 
 . Acquire a Repose Docker image.
-.. Download the Docker image. See: <<../recipes/quick-start.adoc#_repose_docker_images,Repose Docker Images>>
-.. Build the Docker image. See: <<../recipes/docker.adoc#_manually_building_a_repose_docker_image,Manually Building a Repose Docker Image>>
-.. Build the Docker image from the project. See <<Building a Repose Docker Image with the build file>>
-. Run the Repose Docker image in a Docker container. See: <<../recipes/quick-start.adoc#_running_a_docker_container,Running a Docker Container>>
+.. <<../recipes/quick-start.adoc#_repose_docker_images,Download a *Repose* Docker Image>>.
+.. <<../recipes/docker.adoc#_manually_building_a_repose_docker_image,Build a *Repose* Docker Image>>
+.. <<Build a Repose Docker Image with the build file>>
+. <<../recipes/quick-start.adoc#_running_a_docker_container,Run the Repose Docker image in a Docker container>>
 
-== Building a Repose Docker Image with the build file
+== Build a Repose Docker Image with the build file
```

Comment:
Why the change here? I honestly don't have a preference, and I don't know if one is more correct that the other, but now you have two distinct tenses in section headers (i.e., `Running` vs `Build`).